HappyTravel.SupplierRequestLogger/LoggingHandler.cs

```
@@ -66,7 +66,14 @@ private Task Send(HttpRequestAuditLogEntry logEntry)
         {
             using var client = _clientFactory.CreateClient();
             var content = new StringContent(JsonSerializer.Serialize(logEntry), Encoding.UTF8, "application/json");
-            return client.PostAsync(_options.Endpoint, content);
+            try
+            {
+                return client.PostAsync(_options.Endpoint, content);
+            }
+            catch (Exception)
```

1. Swallowing all the exceptions is really bad idea, need to write logs at least
2. Will exception be handled if you are not awaiting?

---

Fixed